packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -262,11 +263,11 @@ export const StripeElement = (props: StripeElementProps) => {
     className,
     autoFocus,
     getString,
+    options = {},
```

Comment:
Don't really need to add this here, either. If options is just passed through to the StripeElementComponent and not used within the wrapper here, it'll just end up in childProps